### ISSUE TITLE
Weird behavior of github for a PR

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2849]
I was working on a new PR (fix #2847) in a new branch (reintroduce_splice_junctions), and when I pushed the branch to open the PR it just merged it to master (which should be impossible, being the branch protected).

When I've tried to revert to the original master (basically undo the commit), I couldn't, because NOW master branch was protected:
![image](https://user-images.githubusercontent.com/28093618/131957441-e102fbe6-56fe-4e97-ab94-343a6478026b.png)

So basically now #2847 is already fixed in master. Could someone check the code and test master, so I can open another PR from the reintroduce_splice_junctions (that in the meanwhile has been pushed to github) to eventually fix what has been merged.

**Comment by northwestwitch** [comment]
Ok, I've managed to revert master. Closing this and opening a proper PR from the other branch!

**Comment by northwestwitch** [comment]
Ok, I've managed to revert master. Closing this and opening a proper PR from the other branch!
